docs/src/tutorials/device-monitoring.md

```diff
@@ -0,0 +1,154 @@
+# Monitor your device with collectd
+
+With thin-edge.io device monitoring, you can collect metrics from your device
+and forward these device metrics to IoT platforms in the cloud. 
+
+Using these metrics, you can monitor the health of devices
+and can proactively initiate actions in case the device seems to malfunction.
+Additionally, the metrics can be used to help the customer troubleshoot when problems with the device are reported.
+
+Thin-edge.io uses the open source component [`collectd`](https://collectd.org/) to collect the metrics from the device.
+Thin-edge.io translates the collected metrics from their native format to the [thin-edge.io JSON](../architecture/thin-edge-json.md) format
+and then into the [cloud-vendor specific format](../architecture/mapper.md).
+
+Enabling monitoring on your device is a 3-steps process:
+1. [Install `collectd`](#install-collectd),
+1. [Configure `collectd`](#configure-collectd),
+1. [Enable thin-edge.io monitoring](#enable-thin-edge-monitoring).
```

Comment:
```suggestion
3. [Enable thin-edge.io monitoring](#enable-thin-edge-monitoring).
```

Reply:
This changes nothing for Markdown. However, fixed.